src/components/WalletBackupForm/WalletBackupForm.js

```diff
@@ -0,0 +1,134 @@
+// @flow strict
+
+import React from 'react'
+import { t } from 'ttag'
+import { Field } from 'react-final-form'
+
+import { fileSaver } from 'services'
+
+import {
+  JIcon,
+  Button,
+  JTextArea,
+  JInputField,
+} from 'components/base'
+
+import walletBackupFormStyle from './walletBackupForm.m.scss'
+
+type Props = {|
+  +handleSubmit: (?SyntheticEvent<HTMLFormElement>) => ?Promise<?Object>,
+  +name: string,
+  +passphrase: ?string,
+  +derivationPath: ?string,
+  +isMnemonic: boolean,
+|}
+
+const BACKUP_MNEMONIC_FIRST_LINE: string = t`Save a wallet backup phrase`
+const BACKUP_MNEMONIC_SECOND_LINE: string = t`to a secure storage or write it down on the paper.`
+
+const BACKUP_KEY_FIRST_LINE: string = t`Save a wallet backup phrase to a secure storage`
```

Comment:
Better for translators to use `<br>` and `dangerouslySetInnerHTML`

Reply:
Why not `\n` ?